contrib/k8s-lbry/README.md

```diff
@@ -0,0 +1,862 @@
+for Kubernetes. [Helm Charts](https://helm.sh/docs/developing_charts/#charts)
+are the basis for packages in Helm. This directory is a Helm chart itself.
+
+All of the helm and kubectl commands necessary to install, upgrade, and maintain
+your deployments, are wrapped in the included [`run.sh`](run.sh) script. For
+debugging purposes, this wrapper also prints to stdout the full underlying
+commands (helm, kubectl, etc) as they are run.
+
+### Create a project directory
+
+Create a new directory someplace to store your deployment configuration. For the
+rest of this tutorial, you will work from this directory:
+
+```
+mkdir $HOME/k8s-lbry-test
+
+cd $HOME/k8s-lbry-test
+```
+
+Download `run.sh` to this same directory:
+
+```
+curl -Lo run.sh https://raw.githubusercontent.com/EnigmaCurry/lbry-docker/k8s-lbry/contrib/k8s-lbry/run.sh
+
+chmod a+x run.sh
+```
+
+### Setup alias and tab completion
+
+`run.sh` can be run directly without any setup. However, without adding it to
+your `PATH`, you need to specify the full path to the script each time. Setting
+a bash alias for `run.sh` is the quickest way of setting up to run from
+anywhere, as well as activating support for bash tab completion.
+
+One time setup to install alias to `$HOME/.bashrc`:
+
+```
+./run.sh setup-alias
+```
+
+It should prompt you if it is OK for the script to edit `$HOME/.bashrc`. Once
+you confirm, close your terminal session, then reopen it.
+
+Verify the new `k8s-lbry` alias to `run.sh` is working:
+
+```
+k8s-lbry kubectl get nodes
+```
+
+Notice that tab completion should work throughout typing the above command.
+
+### k8s-lbry setup
+
+Setup will check for dependencies, update helm repositories, and create an
+initial config file (`values-dev.yaml`).
+
+```
+k8s-lbry setup
+```
+
+### k8s-lbry install-nginx-ingress
+
+An Ingress Controller
+([nginx-ingress](https://github.com/helm/charts/tree/master/stable/nginx-ingress))
+will help you to route outside internet traffic into your cluster. nginx-ingress
+will also help terminate TLS connections (SSL) so that your containers don't
+need to worry about encryption of traffic.
+
+Install nginx-ingress into the `k8s-lbry` namespace:
+
+```
+k8s-lbry install-nginx-ingress
+```
+
+### k8s-lbry install-cert-manager
+
+[cert-manager](https://docs.cert-manager.io/en/latest/index.html) will provide
+TLS certificates (SSL) for your cluster, using [Let's
+Encrypt](https://letsencrypt.org/).
+
+Install cert-manager into the `cert-manager` namespace:
+
+```
+k8s-lbry install-cert-manager
+```
+
+### k8s-lbry install
+
+Once nginx-ingress and cert-manager are installed, the main helm chart can be
+installed. This installs lbrycrd, chainquery, lbrynet, spee.ch, and mysql,
+depending on what you enable in `values-dev.yaml`.
+
+Find the External IP address for your load balancer:
+
+```
+k8s-lbry kubectl get svc nginx-ingress-controller -o wide
+```
+
+If you find a hostname instead of an IP address, this means your load balancer
```

Comment:
Could we not replace part of this with a script that would just use the OS's included `dig` installation to do this?  If that sounds like a proper pain in the ass nevermind but just sounds like a step we can do for them potentially.

Reply:
This whole paragraph is an implementation detail that only affects AWS users (I'm not sure if it affectrs *all* AWS users, or if it was just my config). On DigitalOcean its a non-issue. I will add a comment they can use `dig` themselves. Scripting this away might make this harder to debug I think, plus end-users might have a better idea of how to route via two ip address than me?

Reply:
added note to use dig if available, but otherwise leaving this as-is.